<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Option</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>312</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="envSet">
    <attribute name="title">
     <string>环境</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>193</width>
       <height>188</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>风速</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="windSpeed">
            <property name="inputMethodHints">
             <set>Qt::ImhDialableCharactersOnly</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>m/s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>风向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="windDir"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>° </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>浪高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="waveHeight"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>m  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>浪向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="waveDir"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>° </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>流速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="curSpeed"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>m/s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>流向</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="curDir"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>° </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="dpMode">
    <attribute name="title">
     <string>模式</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>145</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QRadioButton" name="normalDP">
        <property name="text">
         <string>常规动力定位控制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="zpcwDP">
        <property name="text">
         <string>ZPC-W动力定位控制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wopcDP">
        <property name="text">
         <string>WOPC动力定位控制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optDP">
        <property name="text">
         <string>环境最优动力定位控制</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="controlSet">
    <attribute name="title">
     <string>控制器</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>163</width>
       <height>240</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="pidController">
        <property name="text">
         <string>PID控制器</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string> P </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pValue">
            <property name="inputMethodHints">
             <set>Qt::ImhDialableCharactersOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string> I </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="iValue"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string> D </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dValue"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="nmpcController">
        <property name="text">
         <string>NMPC控制器</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string> T </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tValue">
            <property name="inputMethodHints">
             <set>Qt::ImhDialableCharactersOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string> W1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="w1Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string> W2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="w2Value"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string> W3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="w3Value"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="missionSet">
    <attribute name="title">
     <string>任务</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>213</width>
       <height>232</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>起始位置艏向</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>北位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nOrigin">
               <property name="inputMethodHints">
                <set>Qt::ImhDialableCharactersOnly</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>m  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>东位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eOrigin"/>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>m  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>艏向  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="psiOrigin"/>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>° </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>目标位置艏向</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>北位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nTarget">
               <property name="inputMethodHints">
                <set>Qt::ImhDialableCharactersOnly</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>m  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>东位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eTarget"/>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>m  </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>艏向  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="psiTarget"/>
             </item>
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>° </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="otherSet">
    <attribute name="title">
     <string>其他</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>9</y>
       <width>201</width>
       <height>288</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_18">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>环境最优艏向PID参数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>Kp</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="kp">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>Ki</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="ki">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>Kd</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="kd">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>WOPC</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="2">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>虚拟圆半径</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="radius">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>环境估计参数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="1">
            <widget class="QLineEdit" name="k3">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_45">
             <property name="text">
              <string>K3</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="1">
            <widget class="QLineEdit" name="k1">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_41">
             <property name="text">
              <string>K1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="0">
            <widget class="QLabel" name="label_43">
             <property name="text">
              <string>K2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="k2">
             <property name="inputMethodHints">
              <set>Qt::ImhDialableCharactersOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>336</y>
     <width>312</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="applyBtn">
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
